intellij/org.eclipse.xtend.idea.tests/src/org/eclipse/xtend/idea/autoedit/AbstractCStyleLanguageAutoEditTest.xtend

```diff
@@ -555,63 +567,63 @@ abstract class AbstractCStyleLanguageAutoEditTest extends LightToolingTest {
 		configureByText("  /* foo | */")
 
 		myFixture.type('\n')
-		assertState("  /* foo \n   * |\n   */")
+		assertState("  /* foo \n   * | */")
 	}
 
 	def void testMLComments_09() {
 		configureByText("/* foo |*/")
 
 		myFixture.type('\n')
-		assertState("/* foo \n * |*/")
+		assertState("/* foo \n* |*/")
```

Comment:
I think it is a bug in idea, that it doesn't always add the whitespace before the '*'.